infra/terraform/modules/tezos_node/user_data_rolling.tpl

```diff
@@ -68,16 +68,21 @@ rm snapshot.rolling
 cat > health-logs-analysis.sh << 'EOF'
 #!/bin/bash
 
-if [[ $(docker logs ${network}_node_1 2>&1 | grep 'validator.peer: Fetch of operations ' | grep ' timed out' | wc -l) > 0 ]]; then
-  echo "$(date -R) - health-logs-analysis - Warning : dysfunctionment detected. Killing the node"
+if [[ $(curl localhost:8000/chains/main/blocks/head -w %%{time_total} -o /dev/null --silent) > 1 ]]; then
+  echo "$(date -R) - health-logs-analysis - Warning : Dysfunctionment detected. Restarting the node."
+  mkdir /root/.tezos-mainnet/
+  cp -r /.tezos-mainnet/docker-compose.yml /root/.tezos-mainnet/docker-compose.yml
   ${network}.sh stop
+  sleep 10s
```

Comment:
Why a sleep ?

Reply:
Because if you don't stay enough time, the log stay in it (strange bugged status)